sources/packages/web/src/components/fa-application/Section.vue

```diff
@@ -17,10 +19,37 @@
 </template>
 
 <script lang="ts">
+import { computed } from "vue";
+
+enum StyleTypes {
+  primary = "primary",
+  secondary = "secondary",
+  additional = "additional",
+}
+
 export default {
   props: {
     title: String,
     subTitle: String,
+    type: {
+      type: String,
+      default: StyleTypes.primary,
+      validator: (prop: StyleTypes) => Object.values(StyleTypes).includes(prop),
```

Comment:
Not clear where is this being used

Reply:
The type property is used to allow the same section component to be reused in different levels of the applications, as shown below. Is it what you were asking? Please let me know.

![image](https://user-images.githubusercontent.com/61259237/109232354-13b24680-777d-11eb-8ea1-0b0911386a1d.png)